src/components/TextEditor.vue

```diff
@@ -388,11 +418,41 @@ export default {
 				throw new Error('Impossible to execute a command before editor is ready.')
 			}
 		},
+		insertTextBlock(textBlock, addTriggrer = true) {
+			if (addTriggrer) {
+				this.appendToBodyAtCursor('!')
+			}
+			let content = textBlock.content
+			if (!this.html) {
+				const text = new Text('html', content)
+				content = toPlain(text).value
+			}
+			this.editorInstance.execute('insertItem', { content, isHtml: this.html }, '!')
+		},
 	},
 }
 </script>
 
 <style lang="scss" scoped>
+.text-editor--bordered{
```

Comment:
could bordered be a modifier of the existing class `editor-wrapper` to simplify the structure? there are now two classes for one element.